### PR TITLE
doc getting started: change links to github

### PR DESCRIPTION
**Pull request by keachi**
This will fix issue #9.

### CODE REVIEW

**Comment by karras** [comment]
LGTM

File: doc/getting_started.rst [hunk 1]
```diff
@@ -72,7 +72,7 @@ ControlMaster=auto -o ControlPath='~/.ssh/sockets/%C' -o ControlPersist=30s
 -l root $FQDN``). While the connection is established (and 30 seconds
 after that) a socket file in ``~/.ssh/sockets/`` is generated. Ansible will use this
 socket file to connect to the hosts, and doesn't' need to reauthenticate.
-This speeds up ansible operations considerably especially with many hosts.  
+This speeds up ansible operations considerably especially with many hosts.
```
Comment (karras) [inline doc/getting_started.rst]: s/ansible/Ansible

**Review by karras — CHANGES_REQUESTED** [review]

File: doc/getting_started.rst (hunk 1 above)
Comment (karras) [reply doc/getting_started.rst]: Can you please replace all occurrences within the file? :)

**Comment by keachi** [comment]
Please also add your review

**Review by lordfolken — APPROVED** [review]

**Review by lordfolken — APPROVED** [review]

**Review by karras — APPROVED** [review]

**Pull request merged**